src/options.c

```diff
@@ -619,7 +619,7 @@ static void menuKeybinds_genList( unsigned int wid )
    /* Create the list. */
    str = malloc( sizeof( char * ) * input_numbinds );
    for ( j = 0; j < input_numbinds; j++ ) {
-      l = 64;
+      l = 68;
```

Comment:
Why 68?

Reply:
It's somewhat arbitrary. GCC deduces 68 because we have a format string "%s <%s%c>" where "char mod_text[64]" is one of the "%s" args. (That plus brackets plus %c + null gets to 68.)
In practice, we wouldn't expect the message to get that long, and a truncation is only mildly annoying.
Could just double the 64 to 128.